tests/test_datasources.py

```diff
@@ -287,6 +287,7 @@ class SampleData:
             connect_args_dict={
                 'max_download_seconds': '22',
             },
+            expect_identical_connect_args=False,
```

Comment:
Have to excuse this one, 'cause the SQLite max_download_seconds args is digested entirely by postprocess_sqlite, and won't be seen by the Connection layer.